var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -159,6 +161,7 @@ class Sirius(CMakePackage, CudaPackage, ROCmPackage):
     variant(
         "profiler", default=True, description="Use internal profiler to measure execution time"
     )
+    variant("libsci", default=False, description="Use cray-libsci")
```

Comment:
@dev-zero Do you mean something along the lines:

```suggestion
    variant("libsci", default=True, when="+scalapack platform=cray", description="Use cray-libsci")
```

Reply:
@tldahlgren I'd rather not introduce the `libsci` variant at all and later when it is being used in the current patchset, do this instead:

```python
    args = [ "-DUSE_CRAY_LIBSCI=%s" % "ON" if "^cray-libsci" in spec else "OFF" ] 
```

Reply:
because with the current change one has to 1. make sure libsci is being used for blas/lapack (there is no explicit dep on it) and 2. enable the variant. Which seems odd.